src/Raven.Studio/typescript/models/database/cluster/clusterGraph.ts

```diff
@@ -285,6 +296,23 @@ class clusterGraph {
         graphHelper.circleLayout(layoutableNodes, radius);
 
     }
+
+    private updateSize() {
+        this.width = this.$container.innerWidth();
+        this.height = this.$container.innerHeight();
+
+        this.svg.style("width", this.width + "px")
+          .style("height", this.height + "px")
+          .attr("viewBox", "0 0 " + this.width + " " + this.height);
+          
+        this.svg.select("rect")
+            .attr("width", this.width)
+            .attr("height", this.height);
+    }
+
+    removeResizeListener() {
```

Comment:
let's rename this function to `dispose()`. 

This is very common idiom that we use on resources that need cleanup. Then from cluster.ts file we don't care what clean up is needed. We simply call this.graph.dispose(); 

And inside this function we remove event listener. :)